### DISCUSSION TITLE
What are problems with the way we use data and GIS?

### DISCUSSION THREAD

**Discussion by avaitt** [albertkun/21S-ASIAAM-191A#31] [Past Thinking Caps]
Data comes with an inherent bias. This bias can exist in multiple mediums. Bias often starts in data, and a lot number of times algorithms themselves transmit this bias based on the input they receive. An example of this that I can think of has to do with face recognition and the overwhelming number of images/data input that is caucasian. This in turn leads to algorithm bias - with race and gender bias being some of the most obvious consequences of a poorly trained algorithm. Collecting data in ethical ways is also a problem. Instances of famous unethical studies concerning include the case of Henrietta Lacks - a black woman whose cells paved the way for medical research. Henrietta, nor her family, did not receive any praise for her contribution and her name was almost forgotten. Data collection is paramount and geospatial technology software (like GIS) can either worsen the problem or create a solution. Ethical problems concerning GIS software include the fact that it may create stigma associated with certain geographical regions based on some high-level analysis that is generated.

**Comment by okcirE** [comment]
Hey avaitt,

Your response reminded me of an article about implicit bias on pulse oximeters. The bias was caused by ignorant calibration of devices which favored accurate results for lighter skin populations than darker skin populations because devices were calibrated with mostly lighter skin folks. I definitely do agree that bias can often time be based on data input that the program receives, but it is also important to look at other factors that may perpetuate the bias.

**Comment by jsxu315** [comment]
This is a really important point! I know there are some ways to fix this problem on the technical side, such as providing more representative training datasets and designing AI with fairness in mind, but I feel like some of the problem starts with a lack of diversity on the teams that are collecting data/designing algorithms. If teams were more diverse to begin with, maybe we could mitigate some of these problems; since that kind of change happens slowly, maybe bringing on consultants from nonprofits/organizations that are focused on increasing equity could be a stopgap solution.
